src/metrics/family.rs

```diff
@@ -452,4 +469,47 @@ mod tests {
                 .get()
         );
     }
+
+    #[test]
+    fn test_get() {
+        let family = Family::<Vec<(String, String)>, Counter>::default();
+
+        // Test getting a non-existent metric
```

Comment:
```suggestion
        // Test getting a non-existent metric.
```

Nit, convention in the codebase, please also apply below.